dali/base/dasds.cpp

```diff
@@ -2095,6 +2101,146 @@ ISDSManagerServer &querySDSServer()
 
 /////////////////
 
+void CExternalFile::remove(const char *name)
+{
+    StringBuffer filename(name);
+    filename.append(ext);
+    manager.extCache.remove(filename);
+    manager.deltaWriter.removeExt(filename.detach());
+}
+
+void CBinaryFileExternal::readValue(const char *name, MemoryBuffer &mb)
+{
+    StringBuffer extName;
+    getName(extName, name);
+    if (manager.extCache.lookup(extName, mb))
+        return;
+
+    StringBuffer filename;
+    getFilename(filename, name);
+    Owned<IFile> iFile = createIFile(filename.str());
+    size32_t sz = (size32_t)iFile->size();
+    if ((unsigned)-1 == sz)
```

Comment:
Better to cast to size32_t for consistency.

Reply:
will change.